ConsentViewController/Classes/LocalStorage/SPUserDefaults.swift

```diff
@@ -63,10 +66,33 @@ class SPUserDefaults: SPLocalStorage {
         set { storage.setObject(newValue, forKey: SPUserDefaults.LOCAL_STATE_KEY) }
     }
 
+    var gdprChildPMId: String? {
+        get { storage.string(forKey: getChildPmIdKey(type: .gdpr)) }
+        set { storage.set(newValue, forKey: getChildPmIdKey(type: .gdpr)) }
+    }
+
+    var ccpaChildPMId: String? {
+        get { storage.string(forKey: getChildPmIdKey(type: .ccpa)) }
+        set { storage.set(newValue, forKey: getChildPmIdKey(type: .ccpa)) }
```

Comment:
@Nevazhnovu I believe you don't need the method `getChildPmIdKey` since now you know you're storing either CCPA or GDPR child pm id.

Reply:
@andresilveirah do you mean we can store it under the same key? But what if we'll have both GDPR & CCPA `childPmId` included into json response

Reply:
Uh oh I got the idea. Will hardcode both keys and that's it